app/gateway-oss/2.5.x/db-less-and-declarative-config.md

```diff
@@ -204,30 +204,44 @@ parse successful
 
 ## Loading The Declarative Configuration File
 
-There are two ways to load a declarative configuration into Kong: via
-`kong.conf` and via the Admin API.
+There are two ways to load a declarative configuration file into Kong: using
+`kong.conf` or the Admin API.
 
-To load a declarative configuration at Kong start-up, use the
+To load a declarative configuration file at Kong start-up, use the
 `declarative_config` directive in `kong.conf` (or, as usual to all `kong.conf`
 entries, the equivalent `KONG_DECLARATIVE_CONFIG` environment variable).
 
 ```
-$ export KONG_DATABASE=off
-$ export KONG_DECLARATIVE_CONFIG=kong.yml
-$ kong start -c kong.conf
+export KONG_DATABASE=off \
+export KONG_DECLARATIVE_CONFIG=kong.yml \
+kong start -c kong.conf
 ```
 
-Alternatively, you can load a declarative configuration into a running
-Kong node via its Admin API, using the `/config` endpoint. The
+You can also load a declarative configuration file into a running
+Kong node by its Admin API, using the `/config` endpoint. The
```

Comment:
```suggestion
Kong node with the Admin API, using the `/config` endpoint. The
```
this is the form I've been using elsewhere. lmk if you'd like to chat about an alternative.